python/openchange/migration/directory.py

```diff
@@ -235,3 +235,60 @@ def unapply(cls, cur, **kwargs):
                             db.modify_ldif(ldif)
                         except Exception, ex:
                             sys.stderr.write("Error unmigrating user %s: %s\n" % (dn, ex))
+
+
+@migration('directory', 3)
+class AddActiveSyncAttributes(Migration):
+    description = 'Add missing ActiveSync attributes'
+
+    @classmethod
+    def apply(cls, cur, **kwargs):
+        from openchange.provision import get_local_samdb
+        lp = kwargs['lp']
+        creds = kwargs['creds']
+        names = kwargs['names']
```

Comment:
I rather like to unroll the parameters. Also this has been done in the other migrations of the same file.